### PR TITLE
Add missing dot for `/embed` documentation of `csc.exe /?`

### PR DESCRIPTION
**Pull request by dannyvv**

### CODE REVIEW

File: src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf [hunk 1]
```diff
@@ -8295,7 +8295,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
                                但可能被分析器用于生成
                                错误或警告的其他文件。
 -embed                        嵌入 PDB 中的所有源文件。
--embed:&lt;file list&gt;            嵌入 PDB 中的特定文件
+-embed:&lt;file list&gt;            嵌入 PDB 中的特定文件。
```
Comment (dannyvv) [inline src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: I am not familiar with zh-Hans locale. It seems that the `。` character has the same semantics as a period in English.
So I made an educated guess here. Please verify before signing off.

Comment (jaredpar) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: Did you manually edit the file? If so don't do that. Just edit the english file and the build will regenerate these files as needed. A translation team reviews these and follows up with edits ater the fact.

Comment (dannyvv) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: @jaredpar So just the resx and leave all xlf's alone?

Comment (jaredpar) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: Edit the resx, build and it will take care of re-gen the xlf files.

Comment (dannyvv) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: Okay, that was more painfull that it should be 😃 
I've only edited the resx and then `eng\cibuild.cmd` said I should msbuild /t:UpdateXlf or use an env variable.

Attempt 1:
`msbuild /t:UpdateXlf Roslyn.sln` fails with the target doesn't exist:
```
D:\src\github\roslyn\src\Deployment\RoslynDeployment.csproj : error MSB4057: The target "UpdateXlf" does not exist in t
he project.
```
Attempt 2
`set UpdateXlfOnBuild=true` followed by `eng\cibuildcmd` results in:
```
Exception calling "ExtractToDirectory" with "2" argument(s): "The file 'D:\src\github\roslyn\artifacts\Bootstrap\tools\Microsoft.CodeAnalysis.dll' already exists."
System.Management.Automation.MethodInvocationException: Exception calling "ExtractToDirectory" with "2" argument(s): "The file 'D:\src\github\roslyn\artifacts\Bootstrap\tools\Microsoft.CodeAnalysis.dll' already exists." ---> System.IO.IOException: The file 'D:\src\github\roslyn\artifacts\Bootstrap\tools\Microsoft.CodeAnalysis.dll' already exists.
```
had to run a `taskkill -f -im vbcscompiler.exe && git clean -xdf` to get into a good state and rerunning attempt 2 after that resulted in a successful build.

Now all the xlfs are autogenerated by the build.

Comment (nguerrera) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: For future reference: by using `CIBuild.cmd` you're simulating CI which is meant to tell you that the xlf are out of date. UpdateXlfOnBuild will be set up for you when you use build.cmd or build in the IDE.

Comment (dannyvv) [reply src/Compilers/CSharp/Portable/xlf/CSharpResources.zh-Hans.xlf]: Thanks Nick! What is the process to suggest edits the contribution wiki [Contributing Code](https://github.com/dotnet/roslyn/wiki/Contributing-Code)to reflect this? The path for `cibuild.cmd` should be `eng/cibuild.cmd` as well.

**Comment by dannyvv** [comment]
Something went wrong in VSO fetching the repo from github when running the CI's I don't have permissions to rerun the CI's Was hoping someone in the roslyn team could rerun thme.

**Comment by AlekseyTs** [comment]
> Was hoping someone in the roslyn team could rerun them.

Done

**Comment by jcouv** [comment]
Restarted "Windows_Desktop_Unit_Tests debug_64" CI leg again.

**Comment by jcouv** [comment]
Merged. Thanks @dannyvv for the fix.

**Pull request merged**